helpers/operator-payload/build-operators.sh

```
@@ -0,0 +1,343 @@
+    local branch_override="${2:-}"
+    local namespace="$3"
+    local tag="$4"
+    local auth_file="$5"
+    local verbose="$6"
+
+    local operator_name repo_path dockerfile default_branch branch
+
+    operator_name=$(get_config "$operator_alias" "name")
+    repo_path=$(get_config "$operator_alias" "repo")
+    dockerfile=$(get_config "$operator_alias" "dockerfile" "Dockerfile.ocp")
+    default_branch=$(get_config "$operator_alias" "default_branch" "main")
```

Shellcheck (0.11.0)

[warning] default_branch appears unused. Verify use (or export if used externally).

(SC2034)